### PR TITLE
Regal - Partnership Docs

### PR DESCRIPTION
**Pull request by KellieHawks**
@kaiquelupo
Updated submitted documentation to align with other Braze documentation.
Your documentation will be added to the site once we get the go-ahead from our partnership team.
Please feel free to reach out if you have any issues.

Thanks,
The Braze Documentation Team

### CODE REVIEW

File: _docs/_hidden/wip_partnerships/regal.md [hunk 1]
```diff
@@ -0,0 +1,538 @@
...
+| Regal account | A Regal account is required to take advantage of this partnership. |
+| Regal API key | A Regal API key will allow to send events from Braze to Regal.<br><br>Email support@regal.io to get this key. |
+| Braze Data Transformation | Data transformation is currently in early access. Contact your Braze customer success manager if you are interested in participating in the early access. This is necessary to receive data from Regal. |
+{: .reset-td-br-1 .reset-td-br-2}
+
+## Integration: Sending data from Braze to Regal
+
+The following section describes how to use Braze as a source for sending your customer profile and event data to Regal using Braze Canvas webhooks.
+
+### Step 1: Create new contacts in Regal
```
Comment (alex-braze) [inline _docs/_hidden/wip_partnerships/regal.md:33]: We probably want to call out that these could be a Campaign or a Canvas

File: _docs/_hidden/wip_partnerships/regal.md [hunk 2]
```diff
@@ -0,0 +1,538 @@
...
+
+The above payload example assumes all your contacts have accepted opt-in for voice and SMS. If that's not true, you can remove the `optIn` property from the above and set up a separate Canvas to update a contact in Regal when `optIn` is collected.
+
+### Step 2: Update opt-in information 
+
+If opt-in and out can happen at different parts of your user experience on your app, it's important to update Regal as users opt in or out. Below is a recommended Canvas for how to send up-to-date opt-in information to Regal. It assumes you save this as a Braze profile field, but if not, the trigger can just as easily be an event in your Braze account that represents a user opting in or unsubscribing. (The example below is for phone opt-in, but you can set up a similar Canvas for SMS opt-in if you collect those separately).
+
+1. Create a new Canvas titled "Send Opt In or Out to Regal".
+
+2. Set the trigger node logic as "User Profile Field Updated" and select whatever field represents the user's opt-in status. If, you fire an event to Braze to represent opt-in or out, use that event as the trigger instead.
```
Comment (alex-braze) [inline _docs/_hidden/wip_partnerships/regal.md:94]: Although the triggering logic could be "User Profile Field Updated" the customer might want to use the ["Update Subscription Group Status"](https://www.braze.com/docs/user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/triggered_delivery/#setting-up-a-triggered-campaign) trigger or the Subscription Status trigger for global email preference. Which is fired automatically when the subscription status is updated.

**Review by alex-braze — COMMENTED** [review]
Added two comments in:

1. All the webhooks noted in the documentation could be set up as a Canvas or a Campaign so we might want to call that out to ensure customers do not think they have to use a Canvas for each of these use cases.
2. I added a recommendation of another triggering event type the customer might want to use with the subscription preference change webhooks.

File: _docs/_partners/message_orchestration/additional_channels/messaging/regal.md [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+
+Build a Canvas or campaign that webhooks out to Regal every time a new contact is created in Braze who you want to be available for calls and texts in Regal. 
+
+1. Create a Canvas or campaign titled "Create New Contact for Regal" and select **Action-Based** as the entry type.
+
+2. Set the trigger logic as **Custom Event** and select the event that is fired when a contact with a phone number is created. Regal also recommends adding an extra filter on the phone field that ensures it's set.
+
+3. In your new Webhook template, fill out the following fields:<br>
+**Webhook URL**: <https://events.regalvoice.com/events><br>
+**Request Body**: Raw Text
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_orchestration/additional_channels/messaging/regal.md:42]: Formatting suggestion
```suggestion
3. In your new Webhook template, fill out the following fields:
   - **Webhook URL**: <https://events.regalvoice.com/events>
   - **Request Body**: Raw Text
```

File: _docs/_partners/message_orchestration/additional_channels/messaging/regal.md [hunk 4]
```diff
@@ -0,0 +1,540 @@
...
+By integrating Regal and Braze, you can create a more consistent and personalized experience across all your customer touchpoints.
+- Send the right next best email or push notification from Braze based on what's said in a phone conversation on Regal.
+- Trigger a call in Regal when a high-value customer clicks through a marketing email from Braze but doesn't convert.
+
+## Prerequisites
+
+| Requirement | Description |
+| ----------- | ----------- |
+| Regal account | A Regal account is required to take advantage of this partnership. |
+| Regal API key | A Regal API key will allow sending events from Braze to Regal.<br><br>Email support@regal.io to get this key. |
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_orchestration/additional_channels/messaging/regal.md:24]:
```suggestion
| Regal API key | A Regal API key will allow sending events from Braze to Regal.<br><br>Email [support@regal.io](mailto:support@regal.io) to get this key. |
```

File: _docs/_partners/message_orchestration/additional_channels/messaging/regal.md [hunk 5]
```diff
@@ -0,0 +1,540 @@
...
+
+2. Set the trigger logic as **Custom Event** and select the event that is fired when a contact with a phone number is created. Regal also recommends adding an extra filter on the phone field that ensures it's set.
+
+3. In your new Webhook template, fill out the following fields:<br>
+**Webhook URL**: <https://events.regalvoice.com/events><br>
+**Request Body**: Raw Text
+
+#### Request headers and method
+
+Regal.io also requires an HTTP Header for authorization and an HTTP method. The following will already be included within the template as a key-value pair, but in the **Settings** tab:
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_orchestration/additional_channels/messaging/regal.md:46]: Not sure if it makes sense to say "but" here 🤔 I'd suggest striking
```suggestion
Regal.io also requires an HTTP Header for authorization and an HTTP method. The following will already be included within the template as a key-value pair in the **Settings** tab:
```

File: _docs/_partners/message_orchestration/additional_channels/messaging/regal.md [hunk 6]
```diff
@@ -0,0 +1,540 @@
...
+1. Create a new Canvas or campaign titled "Send Opt In or Out to Regal".
+
+2. Select one of the following trigger options and select whatever field represents the user's opt-in status. If you fire an event to Braze to represent opt-in or out, use that event as the trigger instead.
+    - User Profile Field Updated
+    - Update Subscription Group Status
+    - Subscription Status
+
+3. In your new Webhook template, fill out the following fields:<br>
+**Webhook URL**: <https://events.regalvoice.com/events><br>
+**Request Body**: Raw Text
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_orchestration/additional_channels/messaging/regal.md:100]: Same formatting suggestion as above
```suggestion
3. In your new Webhook template, fill out the following fields:
   - **Webhook URL**: <https://events.regalvoice.com/events>
   - **Request Body**: Raw Text
```

File: _docs/_partners/message_orchestration/additional_channels/messaging/regal.md [hunk 7]
```diff
@@ -0,0 +1,540 @@
...
+
+For example, below is a workflow for sending Regal an event when a user completes the first step of an Application.
+
+1. Create a new Canvas or campaign titled "Send Application Step 1 Completed Event to Regal".
+
+2. Set the trigger node logic as **Custom Event** and select the event name you want to send to Regal, such as "Application Step 1 Completed".
+
+3. In your new Webhook template, fill out the following fields:<br>
+**Webhook URL**: <https://events.regalvoice.com/events><br>
+**Request Body**: Raw Text
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_orchestration/additional_channels/messaging/regal.md:150]:
```suggestion
3. In your new Webhook template, fill out the following fields:
   - **Webhook URL**: <https://events.regalvoice.com/events>
   - **Request Body**: Raw Text
```

File: _docs/_partners/message_orchestration/additional_channels/messaging/regal.md [hunk 8]
```diff
@@ -0,0 +1,540 @@
...
+{% alert note %}
+Data transformation is currently in early access. Contact your Braze customer success manager if you are interested in participating in the early access.
+{% endalert %}
+
+Braze recommends creating a transformation per the Regal webhook you plan to send to Braze. 
+
+To create a Data Transformation:
+- Navigate to the Transformations page in your Braze dashboard
+- Give your transformation a name and click **Create transformation**
+- Use the vertical menu icon to copy the webhook URL
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_orchestration/additional_channels/messaging/regal.md:211]:
```suggestion
1. Navigate to the **Transformations** page in your Braze dashboard.
2. Give your transformation a name and click **Create transformation**.
3. From the list of transformations, click <i class="fa-solid fa-ellipsis-vertical" title="View actions"></i> and select **Copy webhook URL**.
```

File: _docs/_partners/message_orchestration/additional_channels/messaging/regal.md [hunk 9]
```diff
@@ -0,0 +1,540 @@
...
+
+![][4]
+
+### Step 2: Enable reporting webhooks in Regal
+
+To set up reporting Webhooks:
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_orchestration/additional_channels/messaging/regal.md:217]:
```suggestion
To set up reporting webhooks:
```

File: _docs/_partners/message_orchestration/additional_channels/messaging/regal.md [hunk 10]
```diff
@@ -0,0 +1,540 @@
...
+
+2. To test the connection, create an outbound call from the Regal Agent Desktop to your cell phone and submit the Conversation Summary form to create a call.completed event.
+
+3. Determine what identifiers you will use to map your Regal contacts to your Braze profiles. The available identifiers in Regal events include:
+   - `userId` - only set on events if you've previously sent this identifier for a contact
+   - `traits.phone`
+   - `traits.email` - only set on events if you've previously sent this identifier for a contact
+
+#### Braze-supported identifiers
+- Braze does not support phone numbers as an identifier. To use this as an identifier, the phone number can be set as a user alias. To learn more about Braze User Aliases, refer to this [documentation][8].
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_orchestration/additional_channels/messaging/regal.md:247]: Suggestion for more meaningful link text
```suggestion
- Braze does not support phone numbers as an identifier. To use this as an identifier, the phone number can be set as a [user alias][8] in Braze.
```

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]
Hey! Looks good, a couple of comments and suggestions for your review :)

**Pull request merged**